extensions/windows-event-log/tests/MetadataWalkerTests.cpp

```diff
@@ -150,10 +141,11 @@ void extractMappingsTestHelper(const std::string &file_name,
   REQUIRE(!input_xml.empty());
   pugi::xml_document doc;
   pugi::xml_parse_result result = doc.load_string(input_xml.c_str());
-  CHECK(result);
+  REQUIRE(result);
 
   auto regex = utils::Regex(".*Sid");
-  MetadataWalker walker(FakeWindowsEventLogMetadata{}, METADATA_WALKER_TESTS_LOG_NAME, update_xml, resolve, &regex);
+  auto user_id_to_username = [](const std::string& user_id) { return utils::OsUtils::userIdToUsername(user_id); };
+  MetadataWalker walker(FakeWindowsEventLogMetadata{}, METADATA_WALKER_TESTS_LOG_NAME, update_xml, resolve, &regex, user_id_to_username);
```

Comment:
is wrapping the `userIdToUsername` in a lambda required?

Reply:
you're right, it isn't -- I have removed it in db3bda60234fcdb1d37aa53f1184b58fce4a2207